doc/services/zbus/index.rst

```diff
@@ -482,23 +765,40 @@ It is possible to add observers to channels in runtime. This feature uses the he
 Samples
 *******
 
-For a complete overview of zbus usage, take a look at the samples. There are the following samples available:
+For a complete overview of zbus usage, take a look at the samples. There are the following samples
+available:
 
 * :zephyr:code-sample:`zbus-hello-world` illustrates the code used above in action;
-* :zephyr:code-sample:`zbus-work-queue` shows how to define and use different kinds of observers. Note there is an example of using a work queue instead of executing the listener as an execution option;
-* :zephyr:code-sample:`zbus-dyn-channel` demonstrates how to use dynamically allocated exchanging data in zbus;
-* :zephyr:code-sample:`zbus-uart-bridge` shows an example of sending the operation of the channel to a host via serial;
-* :zephyr:code-sample:`zbus-remote-mock` illustrates how to implement an external mock (on the host) to send and receive messages to and from the bus.
-* :zephyr:code-sample:`zbus-runtime-obs-registration` illustrates a way of using the runtime observer registration feature;
+* :zephyr:code-sample:`zbus-work-queue` shows how to define and use different kinds of observers.
+  Note there is an example of using a work queue instead of executing the listener as an execution
+  option;
+* :zephyr:code-sample:`zbus-msg-subscriber` illustrates how to use message subscribers.
+* :zephyr:code-sample:`zbus-dyn-channel` demonstrates how to use dynamically allocated exchanging
+  data in zbus;
+* :zephyr:code-sample:`zbus-uart-bridge` shows an example of sending the operation of the channel to
+  a host via serial;
+* :zephyr:code-sample:`zbus-remote-mock` illustrates how to implement an external mock (on the host)
+  to send and receive messages to and from the bus.
+* :zephyr:code-sample:`zbus-runtime-obs-registration` illustrates a way of using the runtime
+  observer registration feature;
+* :zephyr:code-sample:`zbus-confirmed-channel` implements a way of implement confirmed channel only
+  with subscribers.
 * :zephyr:code-sample:`zbus-benchmark` implements a benchmark with different combinations of inputs.
```

Comment:
btw it makes me realize that it would maybe make sense to provide a Sphinx directive that outputs the equivalent of what shows up in the API reference, as this could be useful to list samples anywhere.

<img width="643" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/128251/cbb57472-7692-41ca-af90-c4d7ec40289b">

Reply:
@kartben, how could you change the title of the note? 

![image](https://github.com/zephyrproject-rtos/zephyr/assets/35280/909b023a-d98d-4874-9576-30efee113e59)

Should I remove the Samples sections?

Reply:
Sorry I should have been clearer, and I think it's fine if you live it as-is for now. I was mostly talking to myself and thinking that I should add to the code-sample sphinx extension a way to easily output the list of samples related to a specific set of doxygengroups, as it might be useful in situations like yours. 

That being said, this section pretty much "paraphrases" the list of samples that also already show up automatically in the API reference, mere lines below, so if you think that list is enough and easy enough to not miss for folks, then maybe just drop your Samples section altogether (but again, also 100% fine with me if you'd like to keep it!)